Strawman compatibility categories (#41)
FamilySearch/GEDCOM.io: _pages/compatibility.md

## Compatibility Requirements

* READ/WRITE FAMILYSEARCH GEDCOM FILE
> The vendor needs to demonstrate the ability to both read and write a FamilySearch GEDCOM file.  The [maximal70.ged](..\testfiles\gedcom70\maximal70.ged) file can be used to determine a [compatbility score](https://magikeygedcomconverter.azurewebsites.net/Compatability).

* READ FAMILYSEARCH GEDCOM FILE

> > The vendor only needs to demonstrate the ability to read the [maximal70.ged](..\testfiles\gedcom70\maximal70.ged) file and display the contents in their own environment.

Reviewer: ```suggestion
> The vendor only needs to demonstrate the ability to read the [maximal70.ged](../testfiles/gedcom70/maximal70.ged) file and display the contents in their own environment.
```

> > The vendor only needs to demonstrate the ability to read the [maximal70.ged](../testfiles/gedcom70/maximal70.ged) file and display the contents in their own environment.

Reviewer: ```suggestion
> The vendor needs to demonstrate the ability to read the [maximal70.ged](../testfiles/gedcom70/maximal70.ged) file and display the contents in their own environment.
```

* WRITE FAMILYSEARCH GEDCOM FILE

> > The vendor only needs to demonstrate the ability to write a sample FamilySearch GEDCOM file and prove that it is valid FamilySearch GEDCOM 7. For example, the [GEDCOM validator](http://ged-inline.elasticbeanstalk.com/) can be used, although it is still in beta.

Reviewer: ```suggestion
> The vendor needs to demonstrate the ability to write a sample FamilySearch GEDCOM file and prove that it is valid FamilySearch GEDCOM 7. For example, the [GEDCOM validator](http://ged-inline.elasticbeanstalk.com/) can be used, although it is still in beta.
```

> > The vendor only needs to demonstrate the ability to write a sample FamilySearch GEDCOM file and prove that it is valid FamilySearch GEDCOM 7. For example, the [GEDCOM validator](http://ged-inline.elasticbeanstalk.com/) can be used, although it is still in beta.

Reviewer: ```suggestion
> The vendor only needs to demonstrate the ability to write a sample FamilySearch GEDCOM 7 file and that any FamilySearch GEDCOM 7 files they write are conformant with the FamilySearch GEDCOM 7 specification. For example, the [GEDCOM validator](http://ged-inline.elasticbeanstalk.com/) can be used to check validity, although it is still in beta.
```
